src/Core/System/Country/Aggregate/CountryStateTranslation/CountryStateTranslationCollection.php

```diff
@@ -11,6 +11,9 @@
 #[Package('fundamentals@discovery')]
 class CountryStateTranslationCollection extends EntityCollection
 {
+    /**
+     * @return array<array-key, string>
+     */
     public function getCountryStateIds(): array
```

Comment:
would you mind looking at all the methods in the System collections? most of them are not used, so we should also deprecate them :see_no_evil: